src/test/crimson/test_thread_pool.cc

```diff
@@ -2,16 +2,20 @@
 #include <iostream>
 #include <numeric>
 #include <seastar/core/app-template.hh>
+#include "common/ceph_argparse.h"
+#include "crimson/common/config_proxy.h"
 #include "crimson/thread/ThreadPool.h"
+#include "include/msgr.h"
 
 using namespace std::chrono_literals;
 using ThreadPool = crimson::thread::ThreadPool;
+using crimson::common::local_conf;
 
-seastar::future<> test_accumulate(ThreadPool& tp) {
+seastar::future<> test_accumulate(ThreadPool* tp) {
```

Comment:
nit, could keep this function unchanged.

Reply:
done.